ydb/core/blobstorage/dsproxy/dsproxy_put_impl.h

```diff
@@ -234,6 +247,7 @@ class TPutImpl {
                     p->SetGeneration(generation);
                 }
 
+                History.AddVPut(ptr->Id.PartId(), 1, orderNumber);
```

Comment:
The same thing: it's better to move this code to the single function that processes it all. To prevent losing some queries when other developer adds another place with adding put requests to the outbox.